src/pages/images/actions/UploadCustomIsoBtn.tsx

```diff
@@ -44,6 +48,12 @@ const UploadCustomIsoBtn: FC<Props> = ({ className, project }) => {
         onClick={openPortal}
         className={className}
         hasIcon={!isSmallScreen}
+        disabled={!canCreateStorageVolumes(project)}
+        title={
+          canCreateStorageVolumes(project)
+            ? "Upload custom ISO"
+            : "You do not have permission to create custom ISOs"
```

Comment:
```suggestion
            : "You do not have permission to create custom ISOs in this project."
```
This permission might differ per project.